content/02.intro.md

```diff
@@ -55,28 +55,29 @@ We sketch out a few simple example architectures in Figure @fig:nn-petting-zoo.
-| Recurrent neural network (RNN) | A neural network with cycles between nodes within a hidden layer. This NN architecture is used for sequential data---such as time series or genome sequences. |
-| Long short-term memory (LSTM) neural network | This special type of RNN has features that enable models constructed via this approach to capture longer-term dependencies. |
-| Autoencoder (AE) | A NN where the training objective is to minimize the error between the output layer and the input layer. Such neural networks are unsupervised and are often used for dimensionality reduction. |
-| Variational Autoencoder (VAE) | This special type of AE has the added constraint that the model is trained to learning normally distributed features. |
-| Denoising Autoencoder (DA) | This special type of AE includes a step where noise is added to the input during the training process. |
-| Generative neural network | Neural networks that fall into this class can be used to generate data similar to input data. These models can be sampled to produce hypothetical examples. |
-| Restricted Bolzmann machine (RBM) | Generative NN that forms the building block for many deep learning approaches, having a single input layer and a single hidden layer, with no connections between the nodes within each layer |
-| Deep belief network (DBN) | Generative NN with several hidden layers, which can be obtained from combining multiple RBMs |
-| Generative adversarial network (GAN) | A generative NN approach where two neural networks are trained. One neural network, the generator, is provided with a set of randomly generated inputs and tasked with generating samples. The second, the discriminator, is trained to differentiate real and generated samples. After the two neural networks are trained against each other, the resulting generator can be used to produce new examples. |
-| Adversarial training | A process by which artificial training examples are maliciously designed to fool a NN and then input as training examples to make the resulting NN robust (no relation to GANs) |
-| Data augmentation | A process by which transformations that do not affect relevant properties of the input data (e.g., arbitrary rotations of histopathology images) are applied to training examples to increase the size of the training set. |
+Table @tbl:glossary also provides select example applications, though in practice each neural network architecture has been broadly applied across multiple types of biomedical data.
+A recent book from Goodfellow et al. covers neural network architectures in detail [@url:http://www.deeplearningbook.org/], and LeCun et al. provides a more general introduction [@doi:10.1038/nature14539].
```

Comment:
`provide` instead of `provides`? I think it's based on the many authors as opposed to the single entity paper.